### ISSUE TITLE
Figure out better method for filtering nfts to be shown on marketplace

### ISSUE THREAD

**Issue by rynoV** [mlabs-haskell/seabug#20]
Currently the filtering is done by failing to parse the metadata from blockfrost and some other temporary fixes, see: 
https://github.com/mlabs-haskell/seabug-contracts/blob/a33375d6f2c28bcba00950a912b64dc7569a9e54/src/Seabug/Contract/MarketPlaceListNft.purs#L63

**Comment by samuelWilliams99** [comment]
Yeah, as discussed in the ticket, this will be offloaded to the frontend
